### DISCUSSION TITLE
new file messages.yml

### DISCUSSION THREAD

**Discussion by FrozerYTB** [EssentialsX/Essentials#5414] [Ideas and Feature Suggestions] [answered]
a messages.yml file to change the language of permissions, so you can put it in French or Spanish...

**Comment by FrozerYTB** [comment]
like being able to change the language of /help commands

**Comment by LaxWasHere** [comment, marked as answer]
EssentialsX comes with support for several languages built-in. For more information about changing the EssentialsX language, [click here](https://essentialsx.net/wiki/Locale.html#switching-language).

You can also customise EssentialsX's messages to your liking, from changing colours to editing nearly every message in the plugin. For more information, [click here](https://essentialsx.net/wiki/Locale.html#customising-messages).
